mapwidgets/static/mapwidgets/js/mw_google_point_field_generater.js

```diff
@@ -10,34 +10,33 @@
             var mapOptions = this.mapOptions;
             var widgetData = {};
 
-            var id_regex = new RegExp("(" + prefix + "-(\\d+|__prefix__))");
+            var rowIdSelector = '#'+row[0].id
```

Comment:
sorry link doesnt work

Reply:
@maerzhase I updated the link 👍

Reply:
ok i see the point of doing that but in the original code where are the element id's actually being replaced? i see that the template gets replaced but the ids rendered in the dom are never being touched... 

https://github.com/erdem/django-map-widgets/blob/6fc67b3cc8b2947c5ca5b885cf4b184d405340a7/mapwidgets/static/mapwidgets/js/mw_google_point_field_generater.js#L18-L26

Reply:
```ok i see the point of doing that, but in the original code where are the element id's actually being replaced?```

Yes, the original code also replacing the id attributes. You can see it's replacing when you use the original map widgets. Check the inline row elements.

<img width="798" alt="screen shot 2019-02-05 at 21 50 52" src="https://user-images.githubusercontent.com/1518272/52306569-415aa600-2990-11e9-946d-79a6d2b6180c.png">


I think, the main reason the inline initialization doesn't work because of django admin `inlines.js` loading before the jQuery. I am getting this error from the dev console. 

```
inlines.js:20 Uncaught TypeError: Cannot read property 'fn' of undefined
    at inlines.js:20
    at inlines.js:298
```

The actual django inlines module cannot initialize when the map widget was enabled. 

https://stackoverflow.com/questions/50707750/django-admin-stackedinline-not-loading-add-another-model-uncaught-typeerror-ca

This can be a workaround way, but it is not a proper solution. I'll keep update this PR. 

Thanks.

Reply:
> I think, the main reason the inline initialization doesn't work because of django admin inlines.js loading before the jQuery. I am getting this error from the dev console.

i mentioned in the initial description that changing `(jQuery || django.jQuery)` to `(django.jQuery || jQuery)` makes this error to disappear. can you explain me the point of doing this: `(jQuery || django.jQuery)`? 

> Yes, the original code also replacing the id attributes. You can see it's replacing when you use the original map widgets. Check the inline row elements.

can you point me to the piece of code that does that?

Reply:
> i mentioned in the initial description that changing (jQuery || django.jQuery) to (django.jQuery || jQuery) makes this error to disappear. can you explain me the point of doing this: (jQuery || django.jQuery)?

The error didn't disappear when I switched the jQuery instances order. I got the same error as before. 

`(django.jQuery || jQuery)`

The reason why implement this logic catching Django Admin jQuery triggers. Django Admin using `jQuery` with  `django.jQuery` namespace. You can't reach global `jQuery` variable in original admin templates without adding an additional jQuery library. 

When the users click any "Add a new row" button on admin panel, `inlines.js` jQuery plugin triggering a custom event, which you can see implementation [here](https://github.com/django/django/blob/master/django/contrib/admin/static/admin/js/inlines.js#L126). The inline map widget class implementation has a handler method for add-row event. it's generating a new map widget instance along with new inline row. [Check this line. ](https://github.com/erdem/django-map-widgets/blob/master/mapwidgets/static/mapwidgets/js/mw_google_point_field_generater.js#L6)

So, widget class need to use `django.jQuery.` namespace otherwise we can't get any custom event from django admin. All admin JS implementation triggers using the same namespace.

The event triggering flow is not the main issue; the issue is inline.js loading before admin jQuery.  This wasn't happening before. If there is any static file added to [widget media](https://github.com/erdem/django-map-widgets/blob/master/mapwidgets/widgets.py#L163), `inlines.js` raising that error. Even if I use only a single JS file for widgets, the error shows up.

I am planning to load all media files after page load. That could fix the problem. If you have a better idea feel free to update the topic. Also, let me know if you get any error message during the setup process. 


PS:
> > Yes, the original code also replacing the id attributes. You can see it's replacing when you use the original map widgets. Check the inline row elements.
> 
> 
> can you point me to the piece of code that does that?

You don't need to spend time on this, it is an unrelated point regarding the issue. The original django inlines.js implementation is an example. I shared on the top. 

https://github.com/django/django/blob/master/django/contrib/admin/static/admin/js/inlines.js#L25

Widget class implementation does exactly the same thing as django.

Reply:
@maerzhase FYI, this issue fixed in the latest release. 

https://pypi.org/project/django-map-widgets/0.3.0/